llama31-local-copilot/llama_copilot/prompts.py

```diff
@@ -10,10 +10,4 @@
 - Subject-Specific Jargon: Use industry-specific terms, ensuring they are accessible to a general audience through explanations.
 - Narrative Flow: Ensure a logical flow, connecting ideas and points effectively.
 - Incorporate Statistics and Examples: Support points with relevant statistics, examples, or case studies for real-world context.
-
-## Context
```

Comment:
Why are we removing this?

Reply:
Llama 3.1 doesn't treat the system prompt as a place for context, but rather as a place for instructions.

To address this, we instead [insert context as a separate human message](https://github.com/OpenBB-finance/copilot-for-terminal-pro/pull/23/files#diff-48736d3b523ac261251915290914aa9c148105218e1ecb9c2f482b4bb0c2336fR80). 😊